src/components/projects/projectsList.jsx

```diff
@@ -35,26 +47,40 @@ const projectsList = [
     links: [
       {
         href: 'https://github.com/codeforpdx/PASS',
-        icon: <FaGithub size={45} />
+        icon: <FaGithub />
       },
       {
         href: 'https://discord.gg/x6b573et',
-        icon: <FaDiscord size={45} />
+        icon: <FaDiscord />
       },
       {
         href: 'https://passsmartwallet-967e217a2652.herokuapp.com/',
-        icon: <FaEarthAmericas size={45} />
+        icon: <FaEarthAmericas />
       }
     ],
-    techStack: 'SOLID, React, Vite, JSDocs, MUI, NPM, ES Lint'
+    techStack: ['SOLID', 'React', 'Vite', 'JSDocs', 'MUI', 'NPM', 'ES Lint']
   },
   {
     index: 3,
     title: 'CODE PDX',
     description:
       'Code PDX itself is a project that requires constant upkeep and volunteers to maintain. This ranges from project management, assisting with organizing events, or even updating this website! We are constantly improving to better help our community.',
     logo: '/assets/rose_logo.png',
-    techStack: 'SOLID, React, Vite, JSDocs, MUI, NPM, ES Lint'
+    links: [
+      {
+        href: 'https://github.com/codeforpdx',
+        icon: <FaGithub />
+      },
+      {
+        href: 'https://discord.gg/JcKBq7nXvf',
+        icon: <FaDiscord />
+      },
+      {
+        href: 'https://www.codeforpdx.org/',
+        icon: <FaEarthAmericas />
+      }
+    ],
+    techStack: ['Discord', 'GitHub', 'Google Meets']
```

Comment:
This project is the organization Code PDX, not the website.

Reply:
I think it's a bit weird to have a technology used section, in that case. Especially if there are only a handful of things in it. It's as though we're saying the organization itself has a tech stack the same way the apps do.

However, I'm not sure if I'll like having nothing there any better. Or if it can be replaced with something else entirely.

Reply:
We could take out the techstack array.  I already coded the JSX to be able to handle that as conditional rendering.

Result without it:
<img width="1439" alt="Screenshot 2023-12-22 at 8 45 30 AM" src="https://github.com/codeforpdx/codepdx_website/assets/71395931/616cf884-63c3-4f19-a16e-f85515bf5488">

Reply:
those are in there for constancy across the components and because it lets users know what technologies we use on the business side of things. I think they are fine and its possible this just gets removed once there are more projects to place on the page.